repository-data/application/src/main/resources/hcm-content/content/documents/administration/national-indicator-library/headers.yaml

```diff
@@ -245,6 +257,9 @@
     - headers.interpretationGuidelines
     - headers.qualityStatement
     - headers.technicalSpecification
+    - headers.adviceForm
+    - headers.addForm
+    - headers.applyForm
```

Comment:
Again, remember if you're adding to existing resource bundle this won't have any effect on non-local environments.

Reply:
yep, will add them manually